src/views/pages/ProductTransactions/index.vue

```diff
@@ -10,14 +10,37 @@
 
       <h2 class="error md-subheader">{{error}}</h2>
 
-      <md-table-row slot="md-table-row" slot-scope="{ item }">
+      <md-table-row slot="md-table-row" slot-scope="{ item }" md-selectable="single" md-auto-select>
         <md-table-cell md-label="Date" md-sort-by="datetime">{{item.datetime}}</md-table-cell>
         <md-table-cell md-label="From" md-sort-by="from">{{item.from}}</md-table-cell>
         <md-table-cell md-label="To" md-sort-by="to">{{item.to}}</md-table-cell>
         <md-table-cell :md-label="`Amount (${item.productUnits})`" md-sort-by="amountOfProduct">{{item.amountOfProduct}}</md-table-cell>
         <md-table-cell :md-label="`Rate (${item.productUnits}/${item.currency})`" md-sort-by="costPerUnit">{{item.costPerUnit}}</md-table-cell>
       </md-table-row>
     </md-table>
+
+    <div class="view-dialog-wrapper">
+      <md-dialog v-if="selectedRow" :md-active.sync="showViewDialog">
```

Comment:
Doesn't seem like the `v-if` does anything. I removed it and functionality looked the same.

Reply:
I think it's just a precaution if `selectedRow` was ever empty. Just like checking if object is null.